kythe/go/test/tools/empty_corpus_checker/empty_corpus_checker.go

```diff
@@ -77,5 +81,14 @@ func main() {
 	if emptyCorpusCount != 0 {
 		log.Fatalf("FAILURE: found %d sources with empty corpus", emptyCorpusCount)
 	}
+
+	if len(allowedCorpora) > 0 {
+		allowedCorporaSet := stringset.New(allowedCorpora...)
```

Comment:
Just use flagutil.StringSet rather than converting here?

Reply:
Thanks! I forgot we had that.